src/HasTranslations.php

```diff
@@ -204,6 +205,27 @@ protected function normalizeLocale(string $key, string $locale, bool $useFallbac
         return $locale;
     }
 
+    protected function filterTranslations(string $value = null, string $locale = null, array $allowedLocales = null): bool
```

Comment:
We are having issues with this line, as some translation values aren't strings, 5.0.2 introduces a breaking changes.